tools/install-dummy_hcd.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -xe
+
+apt install linux-source dkms
```

Comment:
I am not sure about this dkms thing. If I understand right, it allows to modprobe the module which automatically loads the dependent modules like `libcomposite` for example. In the beginning I suggest to keep it simple and just use `insmod dummy_hcd.ko`.